### PR TITLE
Improve readme with logo & credits

### PR DESCRIPTION
**Pull request by j0k3r**
Poke @jtojnar.
What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+    <br>
+    <img width="400" height="144" src="https://user-images.githubusercontent.com/62333/67490348-5dfc5280-f673-11e9-9b3d-584e6cbeb9e2.png" alt="Graby logo" />
+    <br>
+    <br>
+    <br>
+    <br>
+</div>
 
-# Graby
+> **Graby** helps you extract article content from web pages
```
Comment (jtojnar) [inline README.md]: I think it makes sense not to duplicate the same paragraph.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
 [![Join the chat at https://gitter.im/j0k3r/graby](https://badges.gitter.im/j0k3r/graby.svg)](https://gitter.im/j0k3r/graby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://travis-ci.org/j0k3r/graby.svg?branch=master)](https://travis-ci.org/j0k3r/graby)
 [![Coverage Status](https://coveralls.io/repos/j0k3r/graby/badge.svg?branch=master&service=github)](https://coveralls.io/github/j0k3r/graby?branch=master)
-[![Total Downloads](https://poser.pugx.org/j0k3r/graby/downloads)](https://packagist.org/packages/j0k3r/graby)
+[![Total Downloads](https://img.shields.io/packagist/dt/j0k3r/graby.svg)](https://packagist.org/packages/j0k3r/graby)
 [![License](https://poser.pugx.org/j0k3r/graby/license)](https://packagist.org/packages/j0k3r/graby)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/j0k3r/graby/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/j0k3r/graby/?branch=master)
 
-Graby helps you extract article content from web pages – created by [Jérémy](https://twitter.com/j0k), logo by [Caneco](https://twitter.com/caneco)
+Graby helps you extract article content from web pages
```
Comment (jtojnar) [inline README.md]: I think, I would prefer this one to remain, since the list below continues from this.

File: README.md (hunk 1 above)
Comment (j0k3r) [reply README.md]: Good point, didn't notice it was also below badges.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,22 @@
...
+<div align="center">
+    <br>
+    <img width="400" height="144" src="https://user-images.githubusercontent.com/62333/67490348-5dfc5280-f673-11e9-9b3d-584e6cbeb9e2.png" alt="Graby logo" />
+    <br>
+    <br>
+    <br>
+    <br>
+</div>
 
-# Graby
```
[the remark this answers is not in the document]
Comment (j0k3r) [reply README.md]: FMPOV, the heading is now the logo, so the `alt` attribute acts as the heading when reading without readme being rendered.

File: README.md (hunk 2 above)
Comment (jtojnar) [reply README.md]: Yup

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jtojnar) [reply README.md]: I am not sure it pops out very much when reading the document in text editor.

**Pull request merged**